src/js/operations/Compress.js

```diff
@@ -346,4 +346,226 @@ var Compress = {
         return plain;
     },
 
+
+    /**
+     * @constant
+     * @default
+     */
+    TAR_FILENAME: "file.txt",
+
+
+    /**
+     * Tar unpack operation.
+     *
+     * @author tlwr [toby@toby.codes]
+     *
+     * @param {byteArray} input
+     * @param {Object[]} args
+     * @returns {byteArray}
+     */
+    runTar: function(input, args) {
+        var zeroFillBytes = function(string, bytes) {
```

Comment:
This should be moved into the `Utils` namespace as it may be useful for other operations. It's similar to [`Utils.padRight`](https://github.com/gchq/CyberChef/blob/master/src/js/core/Utils.js#L74) so put it near that.